packages/commuter-client/src/Notebook.js

```diff
@@ -26,7 +26,7 @@ class Notebook extends React.Component {
   render() {
     return (
       <div>
-        {this.props.isFetching
+        {this.props.isFetching || !this.props.notebook
```

Comment:
Note that I had to do this because sometimes `isFetching` was `false` while `this.props.notebook` was `null`. I'm assuming we need the default state to be an empty notebook or handle this a little differently at first?